### DISCUSSION TITLE
Magnet link

### DISCUSSION THREAD

**Discussion by wullien** [rubo77/AutoIndex#3] [Ideas]
Would it be possible to add the option to add a magnet link folder within the directory and have these links open as a magnet does or be able to upload a magnet into a folder an have a name link and size input option

**Comment by rubo77** [comment]
sure, should be easy to implement, you can create a PR that would be welcome

**Reply by wullien** [reply]
wont allow me to create a pr will find another way to implement this cheers

**Reply by rubo77** [reply]
PRS are allowed

**Reply by wullien** [reply]
They may be allowed but I am not able to do it as it won't allow me to so can't do it as I said I'll figure it out 👍

**Reply by wullien** [reply]
again tried to put a pr up but wont let me so going to try a different script.

**Reply by rubo77** [reply]
In which way "it" won't allow you? is github bocking it? or your tools?

**Reply by rubo77** [reply]
you can just edit it here on github, click on the "edit" icon and it will create a fork in your github account, where you will commit the changes, when done it suggests to create a PR from this commit
